src/Playroom/Playroom.tsx

```diff
@@ -136,8 +135,8 @@ export default ({ components, themes, widths, snippets }: PlayroomProps) => {
   const isVerticalEditor = editorPosition === 'right';
   const isHorizontalEditor = editorPosition === 'bottom';
   const sizeStyles = {
-    height: isHorizontalEditor ? `${editorHeight}px` : 'auto', // issue in ff & safari when not a string
-    width: isVerticalEditor ? `${editorWidth}px` : 'auto',
+    height: isHorizontalEditor ? editorHeight : 'auto', // issue in ff & safari when not a string
```

Comment:
If the height is always a string now given its a percentage, we can remove this comment:
```suggestion
    height: isHorizontalEditor ? editorHeight : 'auto',
```